pages/_document.js

```diff
@@ -9,31 +11,33 @@ const normalizeNextElements = `
     flex-direction: column;
     height: 100%;
   }
-`
+`;
 
 export default class MyDocument extends Document {
-  static async getInitialProps ({ renderPage }) {
-    AppRegistry.registerComponent('Main', () => Main)
-    const { getStyleElement } = AppRegistry.getApplication('Main')
-    const page = renderPage()
+  static getInitialProps({ renderPage }: (cb: Function) => void) {
+    AppRegistry.registerComponent('Main', () => Main);
+    const { getStyleElement } = AppRegistry.getApplication('Main');
+    const page = renderPage();
+
     const styles = [
+      // eslint-disable-next-line react/jsx-key
       <style dangerouslySetInnerHTML={{ __html: normalizeNextElements }} />,
-      getStyleElement()
-    ]
-    return { ...page, styles: React.Children.toArray(styles) }
+      getStyleElement(),
+    ];
+    return { ...page, styles: React.Children.toArray(styles) };
   }
 
-  render () {
+  render() {
     return (
       <html style={{ height: '100%' }}>
```

Comment:
Is it possible to use Stylesheet.create here?

Reply:
I don't know, let's try.

Reply:
Nope, I tried it but it not work. Probably because I tried to apply the react-native style to the DOM element. And I suppose that _document.js use just next.js and react-native don't use it.